src/LibraryManager.Vsix/Shared/Telemetry.cs

```diff
@@ -46,5 +48,114 @@ public static void TrackException(string name, Exception exception)
             string actualName = name.Replace(" ", "_");
             TelemetryService.DefaultSession.PostFault(_namespace + actualName, exception.Message, exception);
         }
+
+        internal static void LogEventsSummary(IEnumerable<ILibraryOperationResult> results, OperationType operation, TimeSpan elapsedTime)
+        {
+            Dictionary<string, object> telResult = new Dictionary<string, object>();
```

Comment:
>Dictionary [](start = 55, length = 10)

This dictionary should use case-insensitive comparison to handle providerIds.

Reply:
ToLower in providers list below.